notebooks/ldb_Feature Engineering_kaggleEX.ipynb

```diff
@@ -0,0 +1,2643 @@
+{
```

Comment:
<p>This don't tell me a lot hahah If is a issue you should put in git. I think you should delte this cell!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done!</p>